.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## What:
```

Comment:
WDYT about adding 'example' to this? So we can see what was wrong/lacked and how it is working now.